x-pack/plugins/observability_solution/metrics_data_access/common/inventory_models/host/metrics/snapshot/rx_new.ts

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { MetricsUIAggregation } from '../../../types';
+export const rxNew: MetricsUIAggregation = {
```

Comment:
I know it will be better to rename the old ones to legacy instead but this will affect the existing alerts (see https://github.com/elastic/kibana/issues/188641#issuecomment-2248451883)